modules/custom-status/lib/custom-status-block.js

```diff
@@ -3,7 +3,7 @@ import './editor.scss';
 import { SelectControl } from '@wordpress/components';
 import { compose } from '@wordpress/compose';
 import { dispatch, select, subscribe, withDispatch, withSelect } from '@wordpress/data';
-import { PluginPostStatusInfo } from '@wordpress/edit-post';
+import { PluginPostStatusInfo } from '@wordpress/editor';
```

Comment:
`PluginPostStatusInfo` from `@wordpress/edit-post` was deprecated in WordPress 6.6 and moved to `@wordpress/editor`. This doesn't appear to have any effect on the component.

Reply:
Would this still be backwards compatible though, since we support all the way back to WordPress 6.0 officially?

Reply:
@ingeniumed Great point! It looks like this change [was added in WordPress 6.6](https://make.wordpress.org/core/2024/06/18/editor-unified-extensibility-apis-in-6-6/) (from [this PR](https://github.com/WordPress/gutenberg/pull/60814)). I'm thinking we should just bump the WordPress supported version to 6.6, as it'll be what VIP customers are largely using, and we may need to use other 6.1+ features in this plugin for workflow management.

Reply:
Did that in https://github.com/Automattic/vip-workflow-plugin/pull/19/commits/5116fefd07db581bf8585adf6ac64d953e3ab26e. Let me know if you think that's a bad idea!

Reply:
I'd lean towards 6.1 or 6.2 instead, not 6.6 imo

Reply:
After internal discussion, I opted to change the minimum down to `6.2` and revert this package name change in https://github.com/Automattic/vip-workflow-plugin/pull/19/commits/2d897915463846a3bc83c32386b358c06218009d. Thanks!